pkg/plugin/twinmaker/client.go

```diff
@@ -534,31 +534,26 @@ func (c *twinMakerClient) GetSessionToken(ctx context.Context, duration time.Dur
 }
 
 func (c *twinMakerClient) GetWriteSessionToken(ctx context.Context, duration time.Duration, workspaceId string) (*sts.Credentials, error) {
-	_, err := c.writerService()
```

Comment:
is this `writerService` call needed? it seems like `GetWriteSessionToken` handles assuming the role

Reply:
I don't think. This was just mirroring the other getsession function really.